go/pkg/client/tree_test.go

```diff
@@ -904,10 +1024,12 @@ func TestComputeMerkleTree(t *testing.T) {
 
 			e, cleanup := fakes.NewTestEnv(t)
 			defer cleanup()
-
+			if tc.treeOpts != nil {
```

Comment:
nit: it is better to do nil check in Apply function?

Reply:
done

Reply:
seems not pushed?

Reply:
https://github.com/bazelbuild/remote-apis-sdks/pull/229/commits/6bdd9894ad2c6daa16b7d743f4c67858a5469647 ?

Reply:
Sorry, I missed that.